mern-ecommerce/routes/order.js

```diff
@@ -29,15 +29,23 @@ router.get(
   requireSignin,
   isAuth,
   isAdmin,
-  getStatusValues
+  getStatus
 );
 
 router.put(
   '/order/:orderId/status/:userId',
   requireSignin,
   isAuth,
   isAdmin,
-  updateOrderStatus
+  updateOrder
+);
+
+router.delete(
```

Comment:
The code snippet provided is incomplete. It only shows the start of a router.delete function, but the rest of the function is missing. Please provide the complete code for a thorough review.